run-cluster.sh

```diff
@@ -1,21 +1,32 @@
 #!/bin/bash
-export K6_TEST_NAME=sc1-proposal-submission-test
+export K6_TEST_NAME=sc1-fap-review-submission-test
 export K6_TEST_FILE="$K6_TEST_NAME.js"
-export K6_VERSION_TAG=0.0.4
-export TEST_SETUP_VERSION_TAG=0.0.4
+export K6_VERSION_TAG=0.0.5
+export TEST_SETUP_VERSION_TAG=0.0.5
 export BROWSER_BASE_URL=https://devproposal.facilities.rl.ac.uk
 export GRAPHQL_URL=https://devproposal.facilities.rl.ac.uk/graphql
 export PROPOSAL_LOOKUP_URL=https://devapis.facilities.rl.ac.uk/ws/ProposalLookupWebService?wsdl
 export TEST_SETUP_URL=http://test-setup:8100
-export K6_PS_VUS=50
-export K6_PS_ITERATIONS=2
-export K6_SETUP_TOTAL_USERS=250
-export TEST_SETUP_CALL_ID=292
-export K6_TEST_PARALLELISM=2
+export SETUP_TEST_REVIEWERS="true"
+export SETUP_TEST_REVIEWER_ROLE="fapMember"
+export K6_PS_VUS=100
+export K6_PS_ITERATIONS=1
+export K6_SETUP_TOTAL_USERS=200
+export TEST_SETUP_CALL_ID=145
+export SETUP_TOTAL_REVIEWERS=100
+export TEST_SETUP_FAP_ID=1
+export TEST_SET_UP_PROPOSAL_PKS=1
+export TEST_SET_UP_INSTRUMENT_ID=37
```

Comment:
We may still need these `export K6_PS_VUS=50
export K6_PS_ITERATIONS=2
export K6_SETUP_TOTAL_USERS=250
export TEST_SETUP_CALL_ID=292
export K6_TEST_PARALLELISM=2`